canary/apps/react/vite/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "vite",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "start": "vite preview --port 3000"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "^2.7.0",
+    "aws-amplify": "^4.3.15",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-react": "^1.0.7",
+    "vite": "^2.8.0"
```

Comment:
This should work to always run the latest 2.x version of Vite, correct?

Reply:
Yes, and we aren't saving the `yarn.lock` so it should be the latest